core/src/test/scala/akka/persistence/cassandra/CassandraSpec.scala

```diff
@@ -52,17 +50,12 @@ object CassandraSpec {
       cassandra-journal {
         session-name = $journalKeyspace
         keyspace = $journalKeyspace
+        # FIXME #81 this is not the way to configure port. Do we need port config in tests?
         port = $port
```

Comment:
?

Reply:
No they all run on port 9042 now with the tests always running against an external c*